internal/tiltfile/files.go

```diff
@@ -120,8 +120,13 @@ func (s *tiltfileState) execLocalCmd(t *starlark.Thread, cmd model.Cmd, options
 		if !options.logOutput {
 			// if we already logged the output, don't include it in the error message to prevent it from
 			// getting output 2x
-			errMessage.WriteString(fmt.Sprintf("\nstdout: %q\nstderr: %q",
-				stdoutBuf.String(), stderrBuf.String()))
+
+			stdout, stderr := stdoutBuf.String(), stderrBuf.String()
+			if strings.Contains(stdout, "\n") || strings.Contains(stderr, "\n") {
+				fmt.Fprintf(&errMessage, "\nstdout:\n%v\nstderr:\n%v\n", stdout, stderr)
+			} else {
+				fmt.Fprintf(&errMessage, "\nstdout: %q\nstderr: %q\n", stdout, stderr)
+			}
```

Comment:
I've made this conditional on "one of the streams has at least one newline" so that any short strings continue to use the old format.

Do you think this is necessary, or could we use this multi-line format all the time? 

I could remove the condition:

```suggestion
			fmt.Fprintf(&errMessage, "\nstdout:\n%v\nstderr:\n%v\n", stdout, stderr)
```